lightning_covid19/model/densenet.py

```diff
@@ -12,13 +12,38 @@
 import torch.nn as nn
 from torch import optim
 from torch.utils.data import DataLoader
+
+import numpy as np
+import kornia as K
 import torchvision.transforms as transforms
 import torchxrayvision as xrv
-from pytorch_lightning.core import LightningModule
 
+from pytorch_lightning.core import LightningModule
 from lightning_covid19.utils import run
 
 
+class KorniaAugmentation(nn.Module):
```

Comment:
Can we rename it to something descriptive? I.e. what the Augmentation does.

You may mention that it uses Kornia augmentation in the docs string.

Reply:
sure, but it's just a generic class to encapsulate the augmentations pipeline to make it easy for anyone to play with different transforms. What about `DataAugmentatorPipe` ?

Reply:
Maybe I am wrong but right now it is not a generic class. It is a particular instance which applies

- RandomHorizontalFlip
- ColorNormalizatoin
- ColorJitter

I see it as a nice example how to use Kornia data augmentation functionalities but not a "API defining" class. Am I wrong?

Reply:
My suggestion is to name it

```
ColorHFlipAugmentation(nn.Module):
    """"An example how to use Kornia data augmentation running RandomHorizontalFlip, Color.Normalize and color jitter""""
```

I want to get this right (and understand it properly) - so it does not create confusion later.

Reply:
More fitting name is welcome - the naming matters :)

Reply:
I guess that the applyed transformations are parameters that can be optimaly set on/off, right? Then I would keep the name a bit general as @edgarriba proposed...

Reply:
@Borda right. Since the project will need different trial and error experiments I think it's good to have this generic class were each of us can enable/disable/add/remove augmentation layers.

Reply:
oh, I understand now